src/wp-includes/default-filters.php

```diff
@@ -603,6 +603,14 @@
 // Global styles custom CSS.
 add_action( 'wp_enqueue_scripts', 'wp_enqueue_global_styles_custom_css' );
 
+// Block template.
+add_filter( 'the_block_template_html', 'shortcode_unautop', 7 );
+add_filter( 'the_block_template_html', 'do_shortcode', 8 );
+add_filter( 'the_block_template_html', 'do_block_template_blocks', 9 );
+add_filter( 'the_block_template_html', 'wptexturize' );
+add_filter( 'the_block_template_html', 'wp_filter_content_tags' );
```

Comment:
Instead of having the order of these two purely defined by the order of these two `add_filter()` calls, I think we should use priority `12` on `wp_filter_content_tags`, also for consistency with how it is intentionally done elsewhere as of https://github.com/WordPress/wordpress-develop/commit/6560f46a0fc5d987901704ba7a676ad685380bc6.
```suggestion
add_filter( 'the_block_template_html', 'wptexturize' );
add_filter( 'the_block_template_html', 'wp_filter_content_tags', 12 );
```